pkg/task/common.go

```diff
@@ -260,6 +266,11 @@ func (cfg *Config) ParseFromFlags(flags *pflag.FlagSet) error {
 	}
 	cfg.CheckRequirements = checkRequirements
 
+	cfg.SwitchModeInterval, err = flags.GetDuration(flagSwitchModeInterval)
+	if err != nil {
+		return errors.Trace(err)
+	}
```

Comment:
add a check to raise an error if the interval is not positive (or disable switch mode when the interval is zero).

Reply:
Ok, add a positive check.